tests/CppUTest/UtestTest.cpp

```diff
@@ -30,6 +30,10 @@
 #include "CppUTest/TestTestingFixture.h"
 #include "CppUTest/PlatformSpecificFunctions.h"
 
+#ifdef __clang__
+# define NEEDS_DISABLE_UNREACHABLE_CODE
```

Comment:
Should this be in the CppUTestConfig instead ? Now it feels a bit like a hack here ?

Reply:
Indeed …
It shouldn't be needed at any other place, but wont hurt in TestConfig.